### PR TITLE
Use the assert/require package in kubelet unit tests

### PR DESCRIPTION
**Pull request by yujuhong**
This reduce the lines of code and improve readability.

### CODE REVIEW

**Comment by cmluciano** [comment]
👍🏽

**Comment by yujuhong** [comment]
go vet was not happy
`W0307 13:48:33.946] pkg/kubelet/kubelet_network_test.go:68: possible formatting directive in Error call`

I changed code slightly to please go vet

File: pkg/kubelet/kubelet_node_status_test.go [hunk 1]
```diff
@@ -670,25 +626,18 @@ func TestUpdateExistingNodeOutOfDiskStatusWithTransitionFrequency(t *testing.T)
 		}
 
 		// Make kubelet report that it has sufficient disk space
-		if err := updateDiskSpacePolicy(kubelet, mockCadvisor, 500, 500, tc.rootFsAvail, tc.dockerFsAvail, 100, 100); err != nil {
-			t.Fatalf("can't update disk space manager: %v", err)
-		}
+		err := updateDiskSpacePolicy(kubelet, mockCadvisor, 500, 500, tc.rootFsAvail, tc.dockerFsAvail, 100, 100)
+		assert.NoError(t, err, "can't update disk space manager")
```
Comment (timstclair) [inline pkg/kubelet/kubelet_node_status_test.go]: require here?

**Review by timstclair — APPROVED** [review]
Much better!

/lgtm

File: pkg/kubelet/kubelet_node_status_test.go (hunk 1 above)
Comment (yujuhong) [inline pkg/kubelet/kubelet_node_status_test.go]: yes, thanks!

**Comment by yujuhong** [comment]
Addressed the comment. Replying the lgtm label based on @timstclair's review above.

**Pull request merged**